tests/trinity/core/p2p-proto/test_stats.py

```diff
@@ -90,8 +90,10 @@ def mk_header_chain(length):
 
         stats = peer.requests.get_stats()
 
-        assert stats['BlockHeaders'].startswith('count={0}, items={0}, avg_rtt='.format(idx))
-        assert stats['BlockHeaders'].endswith(', timeouts=0')
+        header_stats = stats['BlockHeaders']
+        assert header_stats['count'] == idx
+        assert header_stats['items'] == idx
+        assert header_stats['timeouts'] == 0
```

Comment:
You might rebase this on top of #1225 so that we don't conflict here and also the stats available in that branch are easier to get at and probably better.

Reply:
> You might rebase this on top of #1225

Yeah, I just noticed that conflict while reviewing.

> also the stats available in that branch are easier to get at and probably better.

At first glance, I agree. I'll review it now.